doc/versions.json

```diff
@@ -35,6 +36,15 @@
       "tfm": "1.5.0",
       "matter": ""
     },
+    "1.9.2-dev1": {
+      "ncs": "1.9.2-dev1",
+      "nrfxlib": "1.9.99-dev1",
```

Comment:
@gmarull, does this mandate an update in nrfxlib/conf.py?

Reply:
if it changes version, yes